4.x: helidon-config-encryption jar is missing Class-Path

## Environment Details
* Helidon Version: 4.2.6
* Helidon SE 
----------

## Problem Description

`helidon-config-encryption` jar file is missing a `Class-Path` entry in its `MANIFEST.MF`. This means you can't run it without providing a classpath:

```
java -jar target/libs/helidon-config-encryption-4.2.6.jar aes changeit mysecret

Exception in thread "main" java.lang.NoClassDefFoundError: io/helidon/common/config/Config
	at io.helidon.config.encryption.Main$EncryptionCliProcessor.aes(Main.java:116)
	at io.helidon.config.encryption.Main$EncryptionCliProcessor.encrypt(Main.java:103)
	at io.helidon.config.encryption.Main.main(Main.java:45)
Caused by: java.lang.ClassNotFoundException: io.helidon.common.config.Config
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
```

Workaround is to specify classpath:

```
java -cp "target/libs/*" io.helidon.config.encryption.Main aes changeit mysecret
${GCM=xTuF8qYj...}
```

In Helidon 3 the jar file contains a `Class-Path` entry in its `MANIFEST.MF`.

--- a/config/encryption/pom.xml
+++ b/config/encryption/pom.xml
@@ -94,6 +94,7 @@
                 <configuration>
                     <archive>
                         <manifest>
+                            <addClasspath>true</addClasspath>
                             <mainClass>io.helidon.config.encryption.Main</mainClass>
                         </manifest>
                     </archive>

